src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
I have tried to deduce the standard that introduced them... `es5`, `es6`, `html4` (for addEventListener, though that is arguable that is just `dom`).  Since we are only supporting IE9+ and other modern browsers, then I guess the set of prefixes that should be covered off are `es5`, `es6`, `es7`, `html5`, `css3`, `css2` and `dom`.  Can't think of anything else...